### PR TITLE
Taking out upgrade version parameter from write upgrade info

### PR DESCRIPTION
**Pull request by paigerube14**
Taking out the upgrade version parameter because it is no longer needed in my write to sheet script

Please review when you have time @kedark3

### CODE REVIEW

File: Jenkinsfile [hunk 1]
```diff
@@ -23,6 +23,7 @@ pipeline {
         string(name: 'CI_JOB_ID', defaultValue: '', description: 'Scale-ci job id')
         string(name: 'CI_JOB_URL', defaultValue: '', description: 'Upgrade job url')
         string(name: 'LOADED_JOB_URL', defaultValue: '', description: 'Upgrade job url')
+        string(name: 'UPGRADE_VERSION', defaultValue: '', description: 'Versions cluster is upgraded to')
```
Comment (kedark3) [inline Jenkinsfile:26]: Does user pass this param to tell us what version cluster should be upgraded to?

Comment (paigerube14) [reply Jenkinsfile:26]: Yes, the parameter should be set by the user to see what version the cluster should've gotten to. I think I could just  tell that from doing oc commands

**Comment by paigerube14** [comment]
Going to change my script to not expect that parameter because it's not really needed when I can poll the cluster

**Comment by kedark3** [comment]
@paigerube14 can you please also update the PR description ?

**Review by kedark3 — APPROVED** [review]

**Pull request merged**